File: app/src/main/java/nz/co/codebros/quakesnz/list/QuakeListView.java
package nz.co.codebros.quakesnz.list;

import nz.co.codebros.quakesnz.model.Feature;

/**
 * Created by leandro on 9/07/15.
 */
public interface QuakeListView {
    void hideProgress();

    void listQuakes(Feature[] features);

    void showError();

    void showProgress();
}

File: app/src/main/java/nz/co/codebros/quakesnz/list/QuakeListView.kt
package nz.co.codebros.quakesnz.list

import nz.co.codebros.quakesnz.core.model.Feature

/**
 * Created by leandro on 9/07/15.
 */
interface QuakeListView {
    fun listQuakes(features: List<Feature>)

    fun selectFeature(feature: Feature)
}
